src/tests.rs

```diff
@@ -496,7 +463,7 @@ mod governor_tests {
             res.headers()
                 .get(HeaderName::from_static("x-ratelimit-remaining"))
                 .unwrap(),
-            "0" //TODO: Should this be 1?!?
+            "1" //TODO: Should this be 1?!?
```

Comment:
Same as the other test.

Reply:
Same as the other comment, it's fine to change and remove the todo